docs/local-server.md

```diff
@@ -73,16 +73,17 @@ You get the following response:
 | `noxIndex`                        | Number  | Senisirion NOx Index                                                                   |
 | `noxRaw`                          | Number  | NOx raw value                                                                          |
 | `boot`                            | Number  | Counts every measurement cycle. Low boot counts indicate restarts.                     |
-| `bootCount`                       | Number  | Same as boot property. Required for Home Assistant compatability. Will be depreciated. |
+| `bootCount`                       | Number  | Same as boot property. Required for Home Assistant compatability. (deprecated soon!)   |
 | `ledMode`                         | String  | Current configuration of the LED mode                                                  |
 | `firmware`                        | String  | Current firmware version                                                               |
 | `model`                           | String  | Current model name                                                                     |
-| `monitorDisplayCompensatedValues` | Boolean | Switching Display of AirGradient ONE to Compensated / Non Compensated Values           |
 
 Compensated values apply correction algorithms to make the sensor values more accurate. Temperature and relative humidity correction is only applied on the outdoor monitor Open Air but the properties _compensated will still be send also for the indoor monitor AirGradient ONE.
 
 #### Get Configuration Parameters (GET)
-With the path "/config" you can get the current configuration.
+
+With the path "/config" you can get monitor current configurations.
```

Comment:
Propose to rephrase to

"/config" returns the current configuration of the monitor